### PR TITLE
added two new 502 errors

### LINKED ISSUE #358: Give accurate explanations of Pantheon errors and server responses
Story:
As a Pantheon Developer I want current screenshots and accurate explanations of Pantheon errors and server responses so I don't have to file a ticket to find out what they mean

https://pantheon.io/docs/articles/sites/errors-and-server-responses/

Note from Josh: Product should also work to insure that we are as unambiguous as possible.

Work is happening in #623

### PR DESCRIPTION
**Pull request by nataliejeremy**
This resolves issue #370 and #621 
To resolve #358, 
 - [x] Evaluate existing statements for accuracy
 - [x] Fix the statements

### CODE REVIEW

**Comment by nataliejeremy** [comment]
@ari-gold or @timani Can either of you review and +1?

File: source/docs/articles/sites/errors-and-server-responses.md [hunk 1]
```diff
@@ -10,84 +10,115 @@ Sometimes there are problems in the cloud and one of Pantheon's services is unab
...
 ### Pantheon - 404 Unknown Site
-![](/source/docs/assets/images/desk_images/184679.png)  
-"The hostname ... is unknown. Please double-check that this is the right url. If so, make sure it matches your Dashboard's custom domain settings." This typically is shown when there is an internal routing problem or a site environment does not exist.
+"The hostname ... is unknown. Please double-check that this is the right URL. If so, make sure it matches your Dashboard's custom domain settings." This typically is shown when there is an internal routing problem or a site environment does not exist.
+
+### 502 - Upstream Header Too Big
+"Upstream sent too big header while reading response header from upstream."
+This error will occur when the payload or size of the request being sent is greater than the `fastcgi_buffer_size`. Removing additional images reduces the size of the payload being sent to the buffer for nginx to process, and will allow you to post the request. If this happens again, check to see if you are making heavy requests with a number of assets or data being passed.
+
+### 502 - Timeout/Segfault
```
Comment (bmackinney) [inline source/docs/articles/sites/errors-and-server-responses.md]: This section looks like its own document. Anyone else favor clipping this after the first line or two and then linking to a new doc for segfaults?

**Comment by nataliejeremy** [comment]
A whole doc for one error message? I think we should keep them on the page, just my two cents.

File: source/docs/articles/sites/errors-and-server-responses.md [hunk 2]
```diff
@@ -10,84 +10,115 @@ Sometimes there are problems in the cloud and one of Pantheon's services is unab
...
+This issue can happen with HTTP Basic Auth and Drupal’s AJAX, as it doesn't always send the authentication if it is performing an AJAX request.
```
Comment (bmackinney) [inline source/docs/articles/sites/errors-and-server-responses.md]: @timani Is this the only situation in which this happens? Are there others? @nataliejeremy Remove the example ticket content.

**Comment by nataliejeremy** [comment]
@ari-gold @timani Can you review the two new errors?

**Comment by nataliejeremy** [comment]
@rachelwhitton Can you merge this one? I think it's ready to go...

**Comment by bmackinney** [comment]
@timani can you +1?

**Comment by timani** [comment]
+1 enhancements are an upgrade, we can revise later

**Pull request merged**